# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/svg/SVGForeignObjectFrame.cpp
# Commit: a261bb633790
# Full Hash: a261bb633790ddca9cc8ce83adca0a1bb36114a6
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2022-02-23 09:38:00
# Regressor Bug: 1714584
# File Overlap Count: 2
# Description:
#   Bug 1714584 - Part 1: Decouple nsDisplayList internal list from nsDisplayItems r=mstange
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138152
# ==============================================================================

diff -r 635f402efb1b -r a261bb633790 layout/svg/SVGForeignObjectFrame.cpp
--- a/layout/svg/SVGForeignObjectFrame.cpp	Tue Feb 22 23:42:08 2022 +0000
+++ b/layout/svg/SVGForeignObjectFrame.cpp	Tue Feb 22 23:42:18 2022 +0000
@@ -154,7 +154,7 @@
   if (!static_cast<const SVGElement*>(GetContent())->HasValidDimensions()) {
     return;
   }
-  nsDisplayList newList;
+  nsDisplayList newList(aBuilder);
   nsDisplayListSet set(&newList, &newList, &newList, &newList, &newList,
                        &newList);
   DisplayOutline(aBuilder, set);